src/applications/ask-va/containers/DashboardCards.jsx

```diff
@@ -183,7 +162,24 @@ const DashboardCards = () => {
             <div className="vads-u-flex--1 vads-u-margin-left--2">
               <VaSelect
                 hint={null}
-                label="Status"
+                label="Filter by category"
+                name="category"
+                value={categoryFilter}
+                onVaSelect={event => setCategoryFilter(event.target.value)}
+                uswds
```

Comment:
The `uswds` prop is no longer necessary since all design system web components default to the USWDS version now.

Reply:
Noted for the follow up ticket for Dashboard